Makefile

```diff
@@ -195,6 +195,7 @@ update-swagger-docs: statik
 	else \
 		echo "\033[92mSwagger docs are in sync\033[0m";\
 	fi
+	@rm -f ./client/docs/swagger-ui/swagger.yaml
```

Comment:
I don't think we need to commit the file.

Reply:
Ah no, I guess that what being served on /swagger/ rest endpoint.